README.md

```diff
@@ -328,6 +328,12 @@ GET _search
         }
 ```
 
+#### Resources
+
+Not interested in figuring out how to setup Elasticsearch and Kibana? Take a look at [comms-analyzer-toolbox](https://github.com/bitsofinfo/comms-analyzer-toolbox) which bundles
```

Comment:
This section doesn't really belong here, can you please remove it?

Reply:
Relevant, just a simple link to a tool that makes it easier to use the functionality of this project.

Reply:
Sure but I'm not affiliated with that project so I prefer to remove the link.